L/libsndfile/build_tarballs.jl

```diff
@@ -0,0 +1,50 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "libsndfile"
+version = v"1.0.28"
+
+# Collection of sources required to build FLAC
+sources = [
+    "http://www.mega-nerd.com/libsndfile/files/libsndfile-1.0.28.tar.gz" =>
```

Comment:
Very minor nitpicking: use `$(version)` here so that when upgrading the builder to a newer version we can change the version number only in the variable above:
```suggestion
    "http://www.mega-nerd.com/libsndfile/files/libsndfile-$(version).tar.gz" =>
```